packages/roosterjs-editor-api/lib/format/setOrderedListNumbering.ts

```diff
@@ -0,0 +1,35 @@
+import { ChangeSource, IEditor } from 'roosterjs-editor-types';
+import { createVListFromRegion } from 'roosterjs-editor-dom';
+
+/**
+ * Resets Ordered List Numbering back to 1
+ * @param editor The editor instance
+ * @param separator The HTML element that indicates when to split the VList
+ * @param startNumber The number of that the splitted list should start
+ */
+export default function setOrderedListNumbering(
+    editor: IEditor,
+    separator: HTMLLIElement,
+    startNumber: number
+): void;
+export default function setOrderedListNumbering(
+    editor: IEditor,
+    separator: HTMLLIElement,
+    startNumber: number = 1
+) {
+    editor.addUndoSnapshot(() => {
+        editor.focus();
+        const regions = editor.getSelectedRegions();
+        regions.forEach(region => {
+            const vList = createVListFromRegion(region, true, separator /*includeSiblingLists*/);
+            if (vList) {
+                if (separator != vList.items[0].getNode()) {
```

Comment:
Even if separater == vlist.item[0], we can still do this operation. Because the first list item number can be greater than 1, but we can set it to 1, or any other number.